lua/entities/gmod_tardis/modules/sh_thirdperson.lua

```diff
@@ -61,6 +61,10 @@ if SERVER then
             self:CallHook("ThirdPerson", ply, enabled)
         end
     end)
+
+    ENT:AddHook("ThirdPerson", "client", function(self, ply, enabled)
```

Comment:
This seems to be a somewhat common pattern, we should probably build it in with e.g. `ENT:CallSharedHook` / `ENT:CallSharedCommonHook` or similar

Doesn't need to be in this PR just a thought for future ones